ignite/contrib/metrics/MARE.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import division
```

Comment:
nit: maybe it'd be better to rename the file as `mean_absolute_relative_error` instead of `MARE`

Reply:
Sure. It is important to keep consistent. If other metrics use full name, it would be better to change it.

Reply:
Fixed.